CMakeLists.txt

```diff
@@ -681,44 +684,47 @@ endif()
 include(FetchRubyMinGW.cmake)
 FetchRubyMinGW()
 
-# openstudio gems
-# CI will build and upload openstudio gems to s3 at https://github.com/NREL/OpenStudio-gems
-# need to  update the MD5sum for each platform and url below
-if(UNIX)
-   if(APPLE)
-     set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20200622-darwin.tar.gz")
-     set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "4212a009d37e38b087aad51b4b2800e0")
-   else()
-     set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20200622-linux.tar.gz")
-     set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "eb52b76926b82f6c0505fb57d2f46c5b")
-   endif()
-elseif(WIN32)
-  if(CMAKE_CL_64)
-    set(OPENSTUDIO_GEMS_ZIP_FILENAME "openstudio3-gems-20200622-windows.tar.gz")
-    set(OPENSTUDIO_GEMS_ZIP_EXPECTED_MD5 "52c80458909142726b1287dabe0da7a3")
+if(CMAKE_CL_64)
```

Comment:
Looks like this is causing linux and mac to fail as this is windows specific check and the gems packages don't get downloaded. Maybe moved after the UNIX () block?

Reply:
sorry @tijcolem. I will work on a fix right away.

Reply:
I'm debating changing the logic around to avoid the `if (CMAKE_CL_64) ` altogether on non windows platforms, or move to the seemingly cross platform CMAKE_SIZEOF_VOID_P. CMAKE_CL_64 is now "discouraged" according to the documentation. 

I think I'll do a combination. avoid the platform check on non windows. Use the CMAKE_SIZEOF_VOID_P on windows.

For what it is worth, oddly this compiles ok for me on my Mac